### ISSUE TITLE
Unable to clear "APPROVALS" table

### ISSUE THREAD

**Issue by ejm-guden** [cosmocode/dokuwiki-plugin-publish#122]
I created [APPROVALS] tables on several of my wiki pages, all displaying the pages from different namespaces. On each one, I tried to 'approve' all the pages listed on the table--only to find that the last listing on each table refused to clear. The page itself shows as approved, but it still appears on the table (in red). Also, this might be unrelated, but the "approve namespace" button at the top of the table doesn't do anything.
When I got rid of the [APPROVALS] table, saved, then created a /new/ one, it showed "no documents pending approval". While this workaround seems to work just fine, it's definitely not ideal--particularly since I plan to have other employees approving pages. It would definitely float better with them if there were as little fuss as possible.

**Comment by phy25** [comment]
Please read: https://www.dokuwiki.org/caching

TLDR: Add `~~NOCACHE~~` to your page containing `[APPROVALS]`.
